src/ring_theory/polynomial/hermite/basic.lean

```diff
@@ -127,4 +132,78 @@ end
 
 end coeff
 
+section coeff_explicit
+
+open_locale nat
+
+/-- Because of `coeff_hermite_of_odd_add`, every nonzero coefficient is described as follows. -/
+lemma coeff_hermite_explicit :
+  ∀ (n k : ℕ), coeff (hermite (2 * n + k)) k = (-1)^n * (2 * n - 1)‼ * nat.choose (2 * n + k) k
```

Comment:
We usually write `n` and `k` before the colon.

Reply:
Hmm, I'm not sure how to do the case matching correctly here in tactic mode with `induction`.

The awkward thing is that the proof of `(n+1) 0` case reduces to the `n 1` case (which is either the `0 _` or `(n+1) (k+1)` case depending on the value of `n`), while the `(n+1) (k+1)` case uses both the `(n+1) k` and `n (k+2)` cases.

Reply:
I see. It's not very important.